packages/babel-traverse/src/path/family.js

```diff
@@ -67,6 +67,30 @@ export function getSibling(key) {
   });
 }
 
+export function getPrevSibling() {
+  return this.getSibling(this.key - 1);
+}
+
+export function getNextSibling() {
+  return this.getSibling(this.key + 1);
+}
+
+export function forEachNextSibling(callback = Function.prototype) {
```

Comment:
The original issue is https://github.com/babel/babel/issues/5223 by @kangax if you didn't see it